src/components/config_profile/src/profile.cc

```diff
@@ -461,6 +464,7 @@ Profile::Profile()
     , stop_streaming_timeout_(kDefaultStopStreamingTimeout)
     , time_testing_port_(kDefaultTimeTestingPort)
     , hmi_capabilities_file_name_(kDefaultHmiCapabilitiesFileName)
+    , hmi_capabilities_cache_file_name_(kDefaultHmiCapabilitiesCacheFileName)
```

Comment:
Would be great to think a bit about way how to refactor ini file to about so many changes on each parameter add. Don't  take into account now. Just thoughts